components/Card.js

```diff
@@ -0,0 +1,10 @@
+import Link from "next/link.js";
+import styled from "styled-components";
+
+export default function Card({ title, id }) {
+  return (
+    <Link href={`recipes/${id}`} passHref legacyBehavior>
+      <p>{title}</p>
+    </Link>
```

Comment:
this is even better and if you want the Link the be a block element, you can style it that way

```suggestion
    <Link href={`recipes/${id}`}>
     {title}
    </Link>
```